<launch>
    <include file="$(find coax_client)/launch/gumstix_cmvision.launch"/>
	<include file="$(find coax_client)/launch/gumstix_coax_server.launch"/>
	<node pkg="coax_client" type="blob_filter_node" name="blob_filter_node" output="screen"/>
    <node pkg="coax_client" type="blob_pattern_node" name="blob_pattern_node" output="screen"/>
	<!--node pkg="coax_client" type="blob_position_node" name="blob_position" output="screen"/-->
</launch>
